components/PreviousSpeakers.js

```diff
@@ -1,63 +1,37 @@
+import Speaker from "./Speaker";
+
 const PreviousSpeakers = () => (
   <div className="speakers">
     <h1>Previous Speakers</h1>
     <hr />
     <p>See other brave alchemists who shared their knowledge</p>
```

Comment:
To może być spokojnie h2. Zauważ, że jest identycznie wyglądający element w SpeakUpForm, który jest opisany innym tagiem.